events/triggers/update

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+UMBREL_ROOT=$(dirname $(readlink -f ../../$0))
+# UMBREL_ROOT=/home/umbrel
+RELEASE="v$(cat $UMBREL_ROOT/events/signals/update)"
+UMBREL_USER=umbrel
+
+if [ $(whoami) != 'root' ]; then
+    UMBREL_USER=$(whoami)
+fi
+
+echo
+echo "======================================="
+echo "============= OTA UPDATE =============="
+echo "======================================="
+echo "========== Stage: Download ============"
+echo "======================================="
+echo
+
+if [ -z $(grep '[^[:space:]]' $UMBREL_ROOT/events/signals/update) ]; then
+    echo "Empty update signal file. No release version not found."
+    exit 1
+fi
+
+# Make sure an update is not in progres
+if [ -f "$UMBREL_ROOT/statuses/update-in-progress" ]; then 
+    echo "An update is already in progress. Exiting now."
+    exit 2
+fi
+
+echo "Creating lock"
+touch $UMBREL_ROOT/statuses/update-in-progress
+
+# Cleanup just in case there's temp stuff lying around from previous update
+echo "Cleaning up any previous mess"
+[ -d /tmp/umbrel-$RELEASE ] && rm -rf /tmp/umbrel-$RELEASE
+
+# Update status file
+cat <<EOF > $UMBREL_ROOT/statuses/update-status.json
+{"state": "installing", "progress": 10, "description": "Downloading Umbrel $RELEASE"}
+EOF
+
+# Clone new release
+echo "Downloading Umbrel $RELEASE"
+
+cd /tmp/umbrel-$RELEASE
+wget -qO- "https://raw.githubusercontent.com/mayankchhabra/umbrel/$RELEASE/install-box.sh" | sh
+
+cd bin/update
+
+echo "Running update install scripts of the new release"
+for i in {00..99}; do
```

Comment:
Maybe it would be better to loop over `ls *-run.sh` here?

Then we only run exactly the correct amount of times and it works for infinite scripts.

The current solution silently fails with over 100 scripts  and loops 100 times even if there's only one script.